### PR TITLE
make "type" input look consistent with "fields" input

### PR DESCRIPTION
**Pull request by BurningTreeC**

### CODE REVIEW

**Comment by BurningTreeC** [comment]
this PR makes the type input field look like the fields input field, for consistency

**Comment by BurningTreeC** [comment]
bump

File: core/ui/EditTemplate/fields.tid [hunk 1]
```diff
@@ -69,7 +69,7 @@ $value={{{ [<newFieldValueTiddler>get[text]] }}}/>
 <$fieldmangler>
 <div class="tc-edit-field-add">
 <em class="tc-edit">
-<<lingo Fields/Add/Prompt>>&nbsp;
+<<lingo Fields/Add/Prompt>>&nbsp;&nbsp;
```
Comment (pmario) [inline core/ui/EditTemplate/fields.tid]: as I wrote ... I'd prefer a CSS solution

Comment (Jermolene) [reply core/ui/EditTemplate/fields.tid]: I think this is a reasonable quick fix.

**Pull request merged**